scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/task/internal/InternalTask.java

```diff
@@ -1051,6 +1051,7 @@ protected TaskLauncherInitializer getDefaultTaskLauncherInitializer(InternalJob
             tli.setWalltime(wallTime);
         }
         tli.setPreciousLogs(isPreciousLogs());
+        //TODO
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1135)